app.js

```diff
@@ -1,9 +1,15 @@
 const express = require("express");
-const { getApi } = require("./controllers/app.controllers");
+const { getApi, getTopics } = require("./controllers/app.controllers");
 const app = express();
 
 app.use(express.json());
```

Comment:
at this point is this line necessary? When thinking about building up our code using TDD we should follow the principles of writing just enough code to pass our tests, although you may indeed make use of this later on in the sprint its not particularly needed right now :)